test/test_pytree.py

```diff
@@ -147,47 +199,29 @@ def run_test(tup):
         ],
     )
     def test_flatten_unflatten_return_type(self, op):
-        x = torch.randn(3, 3)
-        expected = op(x, dim=0)
+        for pytree_impl in (py_pytree, cxx_pytree):
```

Comment:
We can apply a parametrize decorator for this (one can apply multiple parametrize decorators to functions)